main.go

```diff
@@ -135,6 +135,12 @@ func main() {
 			Usage:   "Change home directory",
 			EnvVars: []string{"PLUGIN_HOME"},
 		},
+		&cli.StringFlag{
+			Name:    "safe-directory",
+			Usage:   "Define safe directories",
+			EnvVars: []string{"PLUGIN_SAFE_DIRECTORY"},
+			Value:   "$CI_WORKSPACE",
```

Comment:
Imho this should be in envvars array instead of value

Reply:
This is how other options are done so for sake of consistency imho that would be best way to do this

Reply:
Ahh I was maybe not clear enough. What I wanted to say is: feel free to go ahead and change things to your liking (you can push commits to this branch)! I think that's more efficient than a multi-round back and forth for such a minor change. 

I am not a go pro and might not get it right on the first try :)

Reply:
changed